### PR TITLE
change logic around deciding when to merge PR

### PR DESCRIPTION
**Pull request by Spartakovic**

### CODE REVIEW

File: pkg/digger/digger.go [hunk 1]
```diff
@@ -247,11 +248,16 @@ func ConvertGithubEventToCommands(event models.Event, impactedProjects []configu
 				})
 			}
 		}
-		return commandsPerProject, nil
+		return commandsPerProject, true, nil
 	case models.IssueCommentEvent:
 		event := event.(models.IssueCommentEvent)
 		supportedCommands := []string{"digger plan", "digger apply", "digger unlock", "digger lock"}
 
+		if requestedProject != nil {
```
Comment (fleroux514) [inline pkg/digger/digger.go:262]: If think `&& requestedProject.Name == impactedProjects[0].Name` is automatically implied if `len(impactedProjects) == 1`

Instead maybe log an error if ` requestedProject.Name != impactedProjects[0].Name` which would mean there is a bug..

Comment (Spartakovic) [reply pkg/digger/digger.go:262]: I wonder if failing fast is still better

Comment (Spartakovic) [reply pkg/digger/digger.go:262]: defensive programming :D

Comment (fleroux514) [reply pkg/digger/digger.go:262]: I understand. There probability of an issue related to this is low.  Let's keep it as is.

**Review by fleroux514 — APPROVED** [review]
LGTM

**Review by veziak — APPROVED** [review]
lgtm

**Pull request merged**